### PR TITLE
take into account that a checksum value may be a tuple of valid checksum in EasyBlock.check_checksums

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: easybuild/framework/easyblock.py [hunk 1]
```diff
@@ -1857,8 +1857,15 @@ def check_checksums_for(self, ent, sub='', source_cnt=None):
             if isinstance(checksum, dict):
                 checksum = checksum.get(fn)
 
-            if not is_sha256_checksum(checksum):
-                msg = "Non-SHA256 checksum found for %s: %s" % (fn, checksum)
+            # take into account that we may encounter a tuuple of valid checksums
+            # (see https://github.com/easybuilders/easybuild-framework/pull/2958)
+            if isinstance(checksum, tuple):
```
Comment (Flamefire) [inline easybuild/framework/easyblock.py]: What about `(algo, checksum)` tuples and `{file:checksums}` dicts?

File: easybuild/framework/easyblock.py [hunk 2]
```diff
@@ -1857,8 +1857,15 @@ def check_checksums_for(self, ent, sub='', source_cnt=None):
             if isinstance(checksum, dict):
                 checksum = checksum.get(fn)
 
-            if not is_sha256_checksum(checksum):
-                msg = "Non-SHA256 checksum found for %s: %s" % (fn, checksum)
+            # take into account that we may encounter a tuuple of valid checksums
```
Comment (Flamefire) [inline easybuild/framework/easyblock.py]: typo: `tuuple`

File: easybuild/framework/easyblock.py (hunk 1 above)
Comment (boegel) [reply easybuild/framework/easyblock.py]: Good point, fixed in ba413cfe3

File: easybuild/framework/easyblock.py (hunk 2 above)
Comment (boegel) [reply easybuild/framework/easyblock.py]: fixed in ba413cfe3, thanks

**Pull request merged**